styles.css

```diff
@@ -0,0 +1,31 @@
+body {
+  text-align: center;
+  max-width: 800px;
```

Comment:
What about screens above 800px width? This would definitely make the page look a bit funny possibly